config/registry_image_pruner/image_pruner/prune_images.py

```diff
@@ -87,11 +108,14 @@ def remove_tags(tags: List[Dict[str, Any]], quay_token: str, namespace: str, nam
         # attestation or sbom image
         if (match := tag_regex.match(tag["name"])) is not None:
             if f"sha256:{match.group(1)}" not in image_digests:
```

Comment:
We don't need to make this check. `sha256-*.(sig|att|sbom|src)` tags should not be deleted as long as the image with digest `match.group(1)` exists in the registry.

Reply:
On the other hand, it could be a nice optimization. If the digest _is_ in the tag list, we don't have to query the API at all.

Reply:
well it is tag list, and that tag is in tag list when not using 'activeTagsOnly', but that also shows all removed tags ....

Reply:
oh right, yes that should handle most cases